app/src/main/java/com/example/footprnt/Discover/Services/YelpService.java

```diff
@@ -59,25 +57,25 @@ public static void findBusinesses(String location, String query, Callback callba
      * Helper function to parse through raw JSON response from Yelp
      *
      * @param response raw response from Yelp API call
-     * @return List of restaurants from Yelp in given location
+     * @return List of businesses from Yelp in given location
      */
     public ArrayList<Business> processResults(Response response) {
+
         ArrayList<Business> businesses = new ArrayList<>();
+
         try {
             String jsonData = response.body().string();
             JSONObject yelpJSON = new JSONObject(jsonData);
             JSONArray businessesJSON = yelpJSON.getJSONArray("businesses");
+
             for (int i = 0; i < businessesJSON.length(); i++) {
                 JSONObject businessJSON = businessesJSON.getJSONObject(i);
                 String name = businessJSON.getString("name");
                 String phone = businessJSON.optString("display_phone", "Phone not available");
                 String website = businessJSON.getString("url");
```

Comment:
I'd really like to see these string keys be class constants so they can be reused and you don't have to worry about spelling mistakes.